mvp_project/README.md

```diff
@@ -1,109 +1,183 @@
+   _Как пользователь, я хочу автоматически записывать маршруты с использованием GPS._  
+   - **Контекст**: Включение GPS для реального времени.  
+   - **Технически**: FastAPI принимает данные через WebSocket или POST-запросы.  
 
-### 10. Рекомендации по маршрутам
-- **Как пользователь**, я хочу получать рекомендации о местах, которые можно посетить, на основе моих интересов, чтобы лучше планировать поездки.
-  - **Контекст:** После загрузки маршрута пользователь видит предложенные направления.
-  - **Технически:** Рекомендательная система использует алгоритмы машинного обучения, анализируя данные других пользователей.
+---
+
+### **Категория 2: Социальные взаимодействия**
+1. **Обмен дневниками**  
+   _Как пользователь, я хочу делиться дневниками через уникальную ссылку._  
+   - **Контекст**: Генерация ссылки с токеном доступа.  
+   - **Технически**: FastAPI предоставляет временный токен доступа.  
+
+2. **Лента активности**  
+   _Как пользователь, я хочу видеть обновления друзей._  
+   - **Контекст**: Лента показывает новые записи и дневники друзей.  
+   - **Технически**: Пагинированный список через FastAPI.  
 
 ---
 
-## Категория 5: Управление приватностью
+### **Категория 3: Интерактивные функции**
+1. **Визуализация маршрутов**  
+   _Как пользователь, я хочу видеть свои маршруты на карте._  
+   - **Контекст**: Карта отображает точки с медиафайлами.  
+   - **Технически**: Использование Google Maps API или Mapbox.  
 
-### 11. Настройки видимости
-- **Как пользователь**, я хочу выбирать, кто может видеть мои дневники, чтобы контролировать доступ к ним.
-  - **Контекст:** В настройках дневника пользователь устанавливает уровень видимости: публичный, только по ссылке, личный.
-  - **Технически:** FastAPI проверяет права доступа при каждом запросе к дневнику.
+2. **Создание видеоисторий**  
+   _Как пользователь, я хочу создавать видеоистории из записей._  
+   - **Контекст**: Выбор шаблона, добавление музыки, экспорт видео.  
+   - **Технически**: Генерация видео через FFmpeg.  
 
 ---
 
-## Категория 6: Геймификация
+### **Категория 4: AI-функции**
+1. **Автоматическое описание мест**  
+   _Как пользователь, я хочу получать описание достопримечательностей на фото._  
+   - **Контекст**: Распознавание объектов на изображениях.  
+   - **Технически**: Google Vision API для анализа.  
 
-### 12. Достижения и награды
-- **Как пользователь**, я хочу получать значки за создание дневников и посещение новых мест, чтобы мотивировать себя.
-  - **Контекст:** Пользователь получает уведомление о новом достижении, например "Посетил 10 стран".
-  - **Технически:** FastAPI фиксирует действия пользователя, обновляет данные и выдаёт награды через систему событий.
+2. **Рекомендации по маршрутам**  
+   _Как пользователь, я хочу получать рекомендации по интересным местам._  
+   - **Контекст**: Система предлагает направления на основе интересов.  
+   - **Технически**: Алгоритмы машинного обучения.  

```

Comment:
Here I would suggest to use something basic for recommendation and later to expand with more advanced ai and recommendation capabilities